InputTime/index.jsx

```diff
@@ -115,20 +117,21 @@ const InputTime = ({
         }
       </select>
       <select
-        disabled={submitting}
+        disabled={disabled || submitting}
         onChange={e => onChange({ ...value, minutes: parseInt(e.target.value, 10) })}
         style={style}
         value={value.minutes}
       >
         {genArray(0, 59).map(min =>
           <option key={min} value={min}>{leftPadTimeUnit(min)}</option>)}
       </select>
-      { select24Hours ? null : renderAmPm({ noStyle, onChange, submitting, value }) }
+      { select24Hours ? null : renderAmPm({ disabled, noStyle, onChange, submitting, value }) }
     </div>
   );
 };
 
 InputTime.propTypes = {
+  disabled: PropTypes.bool.isRequired,
```

Comment:
This will trigger a warning if this isn't set - most of the time this won't be so I don't think this should be required.